modules/generator/processor/spanmetrics/spanmetrics.go

```diff
@@ -79,16 +84,16 @@ func (p *Processor) aggregateMetricsForSpan(svcName string, rs *v1.Resource, spa
 	latencySeconds := float64(span.GetEndTimeUnixNano()-span.GetStartTimeUnixNano()) / float64(time.Second.Nanoseconds())
 
 	labelValues := make([]string, 0, 4+len(p.Cfg.Dimensions))
-	labelValues = append(labelValues, svcName, span.GetName(), span.GetKind().String(), span.GetStatus().GetCode().String())
+	labelValues = append(
```

Comment:
these labels just happen to match up with the order of the dimensions above (which is why this works). i'm fine with this, but maybe we should add a comment to help future us understand that the order here is important.

Reply:
I added a comment explaining the importance of the order of `labelValues`